server/gfmd/db_ldap.c

```diff
@@ -3359,4 +3515,9 @@ const struct db_ops db_ldap_ops = {
 	NULL,
 
 	gfarm_ldap_fsngroup_modify,
+
```

Comment:
move the user_info related entries to just after the gfarm_ldap_user_load entry,
to match the modified order of struct db_ops.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied the above change yet.
Could you change it?